Taskfile.yml

```diff
@@ -55,6 +55,7 @@ tasks:
           OUTPUT_FILE: "{{.CHECKSUM_FILE}}"
 
   emsdk:
+    run: "once"
```

Comment:
This is needed to avoid `emsdk` getting run multiple times when there are other tasks listing `emsdk` as a dependency; however, that is not the case for now. For now, we should remove this to avoid confusions.
```suggestion
```